src/hgnc/index.js

```diff
@@ -227,13 +227,7 @@ const fetchAndLoadBySymbol = async ({
     if (CACHE.SOURCE) {
         hgnc = CACHE.SOURCE;
     } else {
-        hgnc = await conn.addRecord({
-            content: SOURCE_DEFN,
-            existsOk: true,
-            fetchConditions: { name: SOURCE_DEFN.name },
-            fetchExisting: true,
-            target: 'Source',
-        });
+        hgnc = await conn.addSource(SOURCE_DEFN);
```

Comment:
fetchExisting isn't provided by addRecord, is it not actually needed here?

Reply:
fetchExisting can be left off b/c it is default true